### PR TITLE
add_local_asset_for_picture_marker_symbol

### PR DESCRIPTION
**Pull request by sbergmair**
- [x] ⚠️  This branch is based on #56  please review and merge this before

### CODE REVIEW

File: arcgis_map_sdk_android/android/src/main/kotlin/dev/fluttercommunity/arcgis_map_sdk_android/util/GraphicsParser.kt [hunk 1]
```diff
@@ -120,14 +123,40 @@ class GraphicsParser {
         private fun parsePictureMarkerSymbol(map: Map<String, Any>): Symbol {
             val payload = map.parseToClass<PictureMarkerSymbolPayload>()
 
-            return PictureMarkerSymbol(payload.url).apply {
+            // return local asset in case its a local path
+            if(!payload.assetUri.isWebUrl()) {
```
Comment (passsy) [inline arcgis_map_sdk_android/android/src/main/kotlin/dev/fluttercommunity/arcgis_map_sdk_android/util/GraphicsParser.kt:127]: Please apply IntelliJ auto-formatting in kotlin files

File: arcgis_map_sdk_ios/ios/Classes/GraphicsParser.swift [hunk 2]
```diff
@@ -129,7 +136,17 @@ class GraphicsParser {
...
+            let uiImage = getFlutterUiImage(payload.assetUri)
+            let symbol = AGSPictureMarkerSymbol(image: uiImage!)
+            symbol.width = payload.width
+            symbol.height = payload.height
+            symbol.offsetX = payload.xOffset
+            symbol.offsetY = payload.yOffset
+            return symbol
+        }
+
+        let symbol = AGSPictureMarkerSymbol(url: URL(string: payload.assetUri)!)
```
Comment (passsy) [inline arcgis_map_sdk_ios/ios/Classes/GraphicsParser.swift:149]: How is this change related to the `moveCameraToPoints` feature?

File: arcgis_map_sdk_android/android/src/main/kotlin/dev/fluttercommunity/arcgis_map_sdk_android/util/GraphicsParser.kt (hunk 1 above)
Comment (JulianBissekkou) [reply arcgis_map_sdk_android/android/src/main/kotlin/dev/fluttercommunity/arcgis_map_sdk_android/util/GraphicsParser.kt]: ✅
